packages/liburcu/liburcu.spec

```diff
@@ -0,0 +1,67 @@
+Name: %{_cross_os}liburcu
+Version: 0.14.0
+Release: 1%{?dist}
+Summary: Library for userspace RCU
+License: LGPL-2.1-or-later
```

Comment:
Different items in this package are distributed under different licenses, according to the LICENSE file in the sources:

```
The library part is distributed under LGPLv2.1 or later.
.....
.....
xchg() primitive has been rewritten from scratch starting from atomic_ops 1.2
which has a MIT-style license that is intended to allow use in both free and
proprietary software:
....
....
Library test code is distributed under the GPLv2 license. See gpl-2.0.txt for
...
...
The following build-related macro is under GPLv3 (or later):
```

Reply:
Same with this, I will take a look and make sure its correct.